pageserver/src/client/mgmt_api.rs

```diff
@@ -0,0 +1,91 @@
+use anyhow::Context;
+
+use hyper::{client::HttpConnector, Uri};
+use utils::id::{TenantId, TimelineId};
+
+pub struct Client {
+    mgmt_api_endpoint: String,
+    authorization_header: Option<String>,
+    client: hyper::Client<HttpConnector, hyper::Body>,
+}
+
+impl Client {
```

Comment:
There's a little mini-client in control_plane's pageserver.rs -- can you refactor that to use this one to avoid duplication?

Reply:
Sigh, I guess I should 🙄 . Transitioning this PR back into draft state.